tests/php/SubsiteAdminTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace SilverStripe\Subsites\Tests;
+
+use SilverStripe\CMS\Controllers\CMSMain;
+use SilverStripe\Control\Director;
+use SilverStripe\Control\Session;
+use SilverStripe\Security\Member;
+use SilverStripe\Subsites\Model\Subsite;
+
+class SubsiteAdminTest extends BaseSubsiteTest
+{
+    public static $fixture_file = 'subsites/tests/php/SubsiteTest.yml';
+
+    public function adminLoggedInSession()
+    {
+        return new Session([
+            'loggedInAs' => $this->idFromFixture(Member::class, 'admin')
+        ]);
+    }
+
+    /**
+     * Test generation of the view
+     */
+    public function testBasicView()
+    {
+        Subsite::$write_hostmap = false;
+        $subsite1ID = $this->objFromFixture(Subsite::class, 'domaintest1')->ID;
+
+        // Open the admin area logged in as admin
+        $response1 = Director::test('admin/subsites/', null, $this->adminLoggedInSession());
+
+        // Confirm that this URL gets you the entire page, with the edit form loaded
+        $response2 = Director::test("admin/subsites/SilverStripe-Subsites-Model-Subsite/EditForm/field/SilverStripe-Subsites-Model-Subsite/item/$subsite1ID/edit",
+            null,
+            $this->adminLoggedInSession());
+        $this->assertTrue(strpos($response2->getBody(), 'id="Form_ItemEditForm_ID"') !== false,
+            'Testing Form_ItemEditForm_ID exists');
+        $this->assertTrue(strpos($response2->getBody(), '<head') !== false, 'Testing <head> exists');
```

Comment:
These assertions could probably just be `$this->assertContains('<head', (string) $response2->getBody());`